observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc

```diff
@@ -0,0 +1,82 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dist-tracing-tempo-troubleshooting"]
+= Troubleshooting
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-tempo-troubleshooting
+
+toc::[]
+
+You can diagnose and fix TempoStack instance issues by using various troubleshooting methods.
+
+You can inspect the logs of the different components along with some useful metrics.
+
+If the the TempoStack instance has no traces, there are two possible causes:
+
+* TempoStack instance is not ingesting traces.
+* There is an issue with the query path.
+
+[id="getting-tempo-stack-logs_{context}"]
+== Getting the TempoStack instance logs
+
+You can get the logs from a Tempo component as follows.
+
+.Procedure
+
+. Locate the component you want to get the logs for. You can do this by listing all the deployments on an specific namespace.
+
+. Identify the pods that belong to this component.
+
+. Watch the logs by using `oc logs` command or retrieve the logs by using the web console.
+
+[id="problems-ingesting-traces_{context}"]
+== Troubleshooting issues ingesting traces
+
+When the TempoStack instance is failing to ingest traces, you can troubleshoot it as follows.
+
+.Procedure
+
+. Review the components in the ingestion path to diagnose the issue:
++
+* The spans are not sent correctly.
+* The spans are not sampled.
+
+. Check for errors in the logs that identify unhealthy ingesters, which might be failing due to out-of-memory (OOM) issues or scale-down events.
++
+In high-volume tracing environments, the default trace limits might not be adequate.
+If spans are being refused due to these limits, you can discover this in the distributor logs.
++
+You can also see that the `tempo_discarded_spans_total` metric is incremented.
```

Comment:
@rubenvp8510, could you please clarify this in reply in this comment thread? Is the metric incremented a symptom of something? How is it incremented?